practice2.py

```diff
@@ -0,0 +1,87 @@
+print "-----------1-------------------"
+print "Create 1 script to determine is a number is odd or even (use single line statement if applies)"
+
+some_value = 11
+if (some_value%2)==0:
+   print str(some_value)+" is a EVEN value"
+
+else:
+   print str(some_value)+" is a ODD value"
+
+print "----------------2--------------"
+print "According a list of values between a Min and Max range, identify if the number is prime or not."
+
+def is_prime(number):
+    i=1
+    contadorPrimos=0
+    while(i<=number):
+        if(number%i==0):
+            contadorPrimos += 1
+        i+=1
+
+    if (contadorPrimos == 2):
+        return True
+    else:
+        return False
+
+List = [2, 5, 13, 22,50, 100]
+
+for val in List:
+    flag = is_prime(val)
+    if (flag == True):
```

Comment:
please use ternary condition:
`
print str(flag) +(" is prime" if (flag) else " isn't prime")
`